lv_conf_template.h

```diff
@@ -355,7 +355,10 @@
 #define LV_EXPORT_CONST_INT(int_value) struct _silence_gcc_warning /*The default value just prevents GCC warning*/
 
 /*Prefix all global extern data with this*/
-#define LV_ATTRIBUTE_EXTERN_DATA
+#ifndef LV_ATTRIBUTE_EXTERN_DATA
```

Comment:
Do we need the extra `ifndef` guard here?

Reply:
yes because If invoked via cmake LV_ATTRIBUTE_EXTERN_DATA is passed by the command line.
Setting this define will overwrite the command line invocation and resulting in msvc linker errors.
Or do you have another suggestion?